madprops/serializers.py

```diff
@@ -1,19 +1,18 @@
-from collections import Iterable
 import json
 
 from django.db.models import ForeignKey
 from django.utils.functional import cached_property
-from rest_framework.serializers import (
-    ModelSerializer, ModelSerializerOptions, RelationsList)
+from rest_framework.serializers import ModelSerializer, ListSerializer
```

Comment:
I101 Imported names are in the wrong order. Should be ListSerializer, ModelSerializer